app/views/users/index.haml

```diff
@@ -13,10 +13,12 @@
   %tr
     %th Name
     %th Password
+    %th Customer
     %th Sites
 
   - @users.each do |user|
     %tr
       %td= link_to user.name, user
       %td= user.password
-      %td= number_with_delimiter user.sites.count
+      %td= user.customer.present? ? link_to(user.customer.name, customers_path) : ''
```

Comment:
Instead of linking to customer show page here, I linked just to the customers index as this was the behaviour in other areas of the application. So wanted to stay consistent